modules/Bio/EnsEMBL/Hive/RunnableDB/FastaFactory.pm

```diff
@@ -28,6 +28,10 @@
         param('inputfile');         # The original Fasta file: 'inputfile' => 'my_sequences.fasta'
 
         param('max_chunk_length');  # Maximum total length of sequences in a chunk: 'max_chunk_length' => '200000'
+	
+	param('max_chunk_size');    # Defines the maximum allowed number of sequences to be included in each output file.
+	
+	param('seq_filter');        # Can be used to exclude sequences from output files. e.g. '^TF' would exclude all sequences starting with TF.
```

Comment:
Should be spaces, not tabs

Reply:
Done, tabs came from editing the file on the browser.